dev/storage_base.h

```diff
@@ -766,6 +766,9 @@ namespace sqlite_orm {
                 if(this->on_open) {
                     this->on_open(db);
                 }
+                // JD added
+                log_admin::admin().register_db(this, db);
+                // JD end
```

Comment:
same here: this comment is useless cause git blame has info about authors of every line

Reply:
go